### PR TITLE
#61 used saveGzip to accomplish saveRepo task

### PR DESCRIPTION
**Pull request by ammaratef45**
Solved the puzzle in #61 by deleting the body of `saveRepo` and calling for `saveGzip` to accomplish the required task.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#64](https://github.com/artipie/rpm-adapter/issues/64) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/921714f9-8115-472c-9b9e-f49bc9350290); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/rpm/Repomd.java [hunk 1]
```diff
@@ -188,18 +187,9 @@ private CompletableSource performUpdate(
...
-     *  that is closer to the rx paradigm than saveRepo.
      */
     private CompletableSource saveRepo(final Path repomd) {
-        return Completable.fromAction(
-            () -> new BlockingStorage(this.storage).save(
-                new Key.From("repodata/repomd.xml"),
-                Files.readAllBytes(repomd)
-            )
-        );
+        return this.saveGzip(repomd, "repodata/repomd.xml");
```
Comment (Vatavuk) [inline src/main/java/com/artipie/rpm/Repomd.java:192]: @ammaratef45 Let's extract `repodata/repomd.xml` string to a static variable and remove duplication throughout the class.

**Comment by ammaratef45** [comment]
@Vatavuk resolved, please check now

**Comment by Vatavuk** [comment]
@g4s8 ping

**Comment by 0crat** [comment]
@vatavuk[/z](https://www.0crat.com/u/vatavuk) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by g4s8** [comment]
@ammaratef45 thanks for PR, but the design was changed, so it can't be merged

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Pull request [#64](https://github.com/artipie/rpm-adapter/issues/64) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will at least 3 issues during next CR, thanks

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request closed without merging**